src/config/config.cc

```diff
@@ -253,6 +253,8 @@ Config::Config() {
       {"rocksdb.write_options.low_pri", true, new YesNoField(&rocks_db.write_options.low_pri, false)},
       {"rocksdb.write_options.memtable_insert_hint_per_batch", true,
        new YesNoField(&rocks_db.write_options.memtable_insert_hint_per_batch, false)},
+      {"rocksdb.write_options.write_batch_max_bytes", true,
```

Comment:
Can this configuration be modified inflight? if yes, we can change `true` to `false`.

Reply:
You are right, thanks～